api/raft.proto

```diff
@@ -50,6 +51,11 @@ message RaftMember {
 
 	// Status provides the current status of the manager from the perspective of another manager.
 	RaftMemberStatus status = 4 [(gogoproto.nullable) = false];
+
+	// BackupAddrs provides addresses which might be used if main address is
+	// unreachable. This is needed mostly for environments where there might be
+	// short paths with different addresses (nat or whatever).
+	map<string, google.protobuf.Timestamp> backup_addrs = 5;
```

Comment:
Let's go with `AlternateAddrs` as a name.

Any thoughts about also including the primary addr in this map? I feel like that might simplify the code. Eventually we could deprecate `addr`.